pom.xml

```diff
@@ -140,24 +138,27 @@
             <version>2.0</version>
         </dependency>
 
-        <dependency><!-- exists in the core -->
+        <dependency>
             <groupId>org.jenkins-ci.modules</groupId>
             <artifactId>instance-identity</artifactId>
-            <version>1.3</version>
-            <scope>provided</scope>
+            <version>2.1</version><!-- Version matches https://mvnrepository.com/artifact/org.jenkins-ci.main/jenkins-war/2.60.3 -->
+            <scope>provided</scope><!-- https://wiki.jenkins.io/display/JENKINS/Instance+Identity "add a provided scope dependency to this module into your plugin" -->
```

Comment:
What a mess. On my to-do list: make most if not all bundled modules into plain plugins.

Reply:
Just to be clear, you don't want any changes here, right? :)

Reply:
Right, sorry for not being clear—the way modules currently work, there is no clean way to depend on them.